console/commands_as_services.rst

```diff
@@ -70,7 +59,7 @@ To make your command lazily loaded, either define its name using the PHP
     // ...
 
     #[AsCommand(name: 'app:sunshine')]
-    class SunshineCommand extends Command
+    class SunshineCommand
```

Comment:
We reuse this command in the example below to configure it using the `console.command` tag. Is this tag compatible with non-`Command` classes?